configure.ac

```diff
@@ -57,7 +57,7 @@ case "$host" in
 	*-mingw*)
 		CPPFLAGS="${CPPFLAGS} -DWIN32_LEAN_AND_MEAN"
 		CPPFLAGS="${CPPFLAGS} -D_WIN32_WINNT=NTDDI_WINXP"
-		LDFLAGS="${LDFLAGS} -Wl,--nxcompat"
+		LDFLAGS="${LDFLAGS} -Wl,--nxcompat,--dynamicbase,--export-all-symbols"
```

Comment:
Sadly yes, I checked in vmmap & procexp.

I also notice the shipped openssl dll doesn't enable aslr.

Another thought, why this project doesn't support building in MSVC? It supports all these security features flawlessly.